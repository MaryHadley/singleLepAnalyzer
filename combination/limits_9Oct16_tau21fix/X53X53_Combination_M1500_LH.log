000 (1295369.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295369.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.230:40000>
...
006 (1295369.000.000) 10/09 22:41:34 Image size of job updated: 188484
	86  -  MemoryUsage of job (MB)
	87780  -  ResidentSetSize of job (KB)
...
006 (1295369.000.000) 10/09 22:46:34 Image size of job updated: 282760
	144  -  MemoryUsage of job (MB)
	147072  -  ResidentSetSize of job (KB)
...
006 (1295369.000.000) 10/10 00:01:41 Image size of job updated: 282760
	144  -  MemoryUsage of job (MB)
	147084  -  ResidentSetSize of job (KB)
...
006 (1295369.000.000) 10/10 21:33:57 Image size of job updated: 297248
	157  -  MemoryUsage of job (MB)
	159980  -  ResidentSetSize of job (KB)
...
005 (1295369.000.000) 10/10 21:39:12 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 22:51:14, Sys 0 00:04:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 22:51:14, Sys 0 00:04:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1594  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1594  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31850        1   3824811
	   Memory (MB)          :      157     3072      3072
...
