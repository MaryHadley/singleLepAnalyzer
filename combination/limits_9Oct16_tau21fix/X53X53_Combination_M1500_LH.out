/var/lib/condor/execute/dir_9798
Total number of likelihood evaluations: 1100000101
Total number of likelihood evaluations: 11000101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1500_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1500_LH/quant-toys:0-sig-067147b746.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1500_LH/quant-data-sig-f6ca0be1ba.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1500_LH
