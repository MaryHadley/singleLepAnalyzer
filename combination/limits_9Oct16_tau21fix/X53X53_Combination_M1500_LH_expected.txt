# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.077303   0.039148       0.16  0.053807    0.11341
